# Inertial parameters referred to Denavit-Hartenberg parametrization to use regressor
# Obtained with transformation of URDF files considering Denavit-Hartenberg parametrization
# 

link1:
  mass: 4.9706840000000003
  m_CoM_x: 0.019261400500000001
  m_CoM_y: 0.010343993404000001
  m_CoM_z: -0.23670397208000002
  Ixx: 0.71466336900072336
  Ixy: -0.0001790829744405
  Ixz: 0.0076892278918100007
  Iyy: 0.71795648107738708
  Iyz: 0.019661580965898477
  Izz: 0.0092131637772112241

link2:
  mass: 0.646926
  m_CoM_x: -0.002031994566
  m_CoM_y: -0.018579714719999998
  m_CoM_z: 0.0022610063699999997
  Ixx: 0.0085035116240215498
  Ixy: -0.0039833588839355196
  Ixz: 0.010261101821008169
  Iyy: 0.028124284712194955
  Iyz: 0.00076893610294640001
  Izz: 0.026534991901690206

link3:
  mass: 3.2286039999999998
  m_CoM_x: 0.088844724872
  m_CoM_y: 0.12672916420800001
  m_CoM_z: -0.21470862320799999
  Ixx: 0.056494926014070829
  Ixy: -0.0082483331406757437
  Ixz: -0.0054876481065622559
  Iyy: 0.052878381999606108
  Iyz: -0.0043772571218395843
  Izz: 0.018249202292520111

link4:
  mass: 3.5878950000000001
  m_CoM_x: -0.19076837715
  m_CoM_y: 0.37464440800499998
  m_CoM_z: 0.098502069329999997
  Ixx: 0.067677270250859914
  Ixy: 0.02771584317362585
  Ixz: 0.0039053550262761003
  Iyy: 0.032399430424451321
  Iyz: -0.0016444875773692705
  Izz: 0.077586149052539605

link5:
  mass: 1.225946
  m_CoM_x: -0.014653732538
  m_CoM_y: 0.050343472489999999
  m_CoM_z: -0.047121686401999999
  Ixx: 0.039427570958035521
  Ixy: -0.0015152444733270299
  Ixz: -0.0046002455175631054
  Iyy: 0.031460372325260388
  Iyz: 0.0021640520520981297
  Izz: 0.010869510762828563

link6:
  mass: 1.666555
  m_CoM_x: 0.10024161669500001
  m_CoM_y: -0.023526756935
  m_CoM_z: -0.017527158935000002
  Ixx: 0.0024804603581707902
  Ixy: 0.0015241109028833149
  Ixz: -0.00010375891721868492
  Iyy: 0.010567766133106952
  Iyz: 9.3569097314604994e-05
  Izz: 0.011794560230238952

link7:
  mass: 1.66560124
  m_CoM_x: 0.048635556208000003
  m_CoM_y: 0.011492648555999999
  m_CoM_z: -0.075
  Ixx: 0.015436298654092
  Ixy: -0.00076358533783519992
  Ixz: 0.00081264847139040025
  Iyy: 0.0142881576203292
  Iyz: -0.00026635361463719996
  Izz: 0.0063144575163099998

link8:
  mass: 0.940
  m_CoM_x: 0.07
  m_CoM_y: 0
  m_CoM_z: 0
  Ixx: 0.001
  Ixy: 0
  Ixz: 0
  Iyy: 0.0031500000000000005
  Iyz: 0
  Izz: 0.0031500000000000005
